src/radical/pilot/compute_unit_description.py

```diff
@@ -337,7 +339,7 @@ class ComputeUnitDescription(ru.Description):
     """
 
     _schema = {
-               EXECUTABLE      : str         ,
+               EXECUTABLE      : None        ,
```

Comment:
I would like to discuss this change on the next devel call.  Motivation is understood, but it removes type checking from the only required task attribute...

Reply:
I missed the last development call, any update regarding this please?

Reply:
No, that was not yet discussed.  I don't mind merging this now and to start the discussion concurrently.